docs/book/src/migration/project/v2_v3.md

```diff
@@ -8,20 +8,22 @@ to install the required components.
 
 ## Steps to migrate
 
-<aside class="note warning">
+<aside class="note attention">
 <h1>Note</h1>
 
-The following steps will be based on the [QuickStart][QuickStart] steps.
+This guide ONLY describe the steps required to change to update PROJECT file to use the version 3. It is recommended to upgrade your project by following the [Migration Guide v2 to V3][migration-guide-v2-to-v3] instead which is the most assertive and straightforward way.
```

Comment:
nit: `describes`

Reply:
also "most assertive" do you mean "most direct" here?  I'm not certain "most assertive" is quite the right wording